A job failed! Check out the build log: [(web)](https://github.com/rust-lang/rust/actions/runs/13375943219/job/37355130854) [(plain)](https://github.com/rust-lang/rust/commit/d90c50c5818b61f4390bae1539a2c7a9e4b83b63/checks/37355130854/logs)

<details><summary><i>Click to see the possible cause of the failure (guessed by this bot)</i></summary>

```plain
  PR_CI_JOB: 1
  READ_ONLY_SRC: 0
  IMAGE: x86_64-gnu-llvm-18
##[endgroup]
error: PRs changing the `beta` channel should be sent to the `beta` branch!
##[error]Process completed with exit code 1.
```

</details>

--- a/src/ci/channel
+++ b/src/ci/channel
@@ -1,1 +1,1 @@
-nightly
+beta

